src/demo/containers/playground/FeaturedDataTabsDemo.tsx

```diff
@@ -1,70 +1,95 @@
+import { SynapseConstants } from '../../../lib'
 import * as React from 'react'
 import FeaturedDataTabs from '../../../lib/containers/home_page/featured-data/FeaturedDataTabs'
 
 export const FeaturedDataTabsDemo = () => {
   return (
-    <FeaturedDataTabs rgbIndex={0}
-      sql='select * from syn11346063'
-      explorePagePath='/Explore/Studies'
-      exploreObjectType='Studies'
-      configs={
-        [
-          {
-            title: 'Human Studies',
-            icon: 'PERSON',
-            plotsConfig: {
-              configs: [{
-                title:'The Religious Orders and Memory and Aging Project Study',
-                facetsToPlot:['dataType', 'assay'],
-                selectFacetColumnName:'study',
-                selectFacetColumnValue:'ROSMAP',
-                detailsPagePath:'/Explore/Studies/DetailsPage?Study=syn3219045'
+    <>
+      <FeaturedDataTabs rgbIndex={0}
+        sql='select * from syn11346063'
+        configs={
+          [
+            {
+              title: 'Human Studies',
+              icon: 'PERSON',
+              explorePagePath:'/Explore/HumanStudies',
+              exploreObjectType:'Studies',
+              plotsConfig: {
+                configs: [{
+                  title:'The Religious Orders and Memory and Aging Project Study',
+                  facetsToPlot:['dataType', 'assay'],
```

Comment:
Example where each card shows multiple plots.  The query is modified to have a single selected facet value.  A query is run for each facet plots card.